src/ros_license_toolkit/checks.py

```diff
@@ -44,12 +45,17 @@ def __init__(self: 'Check'):
 
     def _failed(self, reason: str):
         """Set this check as failed for reason `r`."""
```

Comment:
```suggestion
        """Set this check as failed for `reason`."""
```
Please also check the other similar methods for this. This may be a thing I changed previously.